array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const doubled = numbers.map(x => x * 2);
+console.log('doubled value: ', doubled);
+
+const prices = numbers.map(x => '$' + x + '.' + '00');
```

Comment:
While this works here it would be better to use `$${number.toFixed(2)}`, that will account for if the number already has a decimal.